planetcantile/data/generate.py

```diff
@@ -132,15 +132,14 @@ def __post_init__(self):
         pass
         #print(f'Could not find authority for {crs_obj.to_wkt()}')
 
-
-for tmsp in crss:
+for tmsp in crss[10:]:
```

Comment:
why was this line changed

Reply:
It shouldn't be. I was seeing errors in crss>10. I'll revert.